Js/main.js

```diff
@@ -0,0 +1,84 @@
+/* eslint func-names: ["error", "never"] */
```

Comment:
- [ ] Kindly remove this disabled rule.

Reply:
Removed  /* eslint func-names: ["error", "never"] */  comment as suggested